packages/lesswrong/components/posts/PostsItemMeta.jsx

```diff
@@ -43,7 +43,7 @@ const PostsItemMeta = ({classes, currentUser, post, read}) => {
 
       { post.isEvent && <MetaInfo>
         {post.startTime
-          ? <Tooltip title={<EventTime post={post} />}>
+          ? <Tooltip title={<Components.EventTime post={post} />}>
```

Comment:
why?

Reply:
See conversation here: https://github.com/LessWrong2/Lesswrong2/pull/2413
Sorry I should have added the same context to this PR.